backend/backend/public_urls.py

```diff
@@ -51,6 +50,8 @@
     import pluggable_apps.platform_admin.urls  # noqa # pylint: disable=unused-import
     import pluggable_apps.public_shares_share_controller.urls  # noqa # pylint: disable=unused-import
 
+    share_path_prefix = settings.PUBLIC_PATH_PREFIX
```

Comment:
@harini-venkataraman what if this is not defined in the settings file (`base.py` or in the cloud version) - is the error handled gracefully?

Reply:
@chandrasekharan-zipstack Have habdled the AttributeError in public_urls.py. If the value is not defined in settings file ( in case of OSS ) we supress the exception and in case of cloud, we have the value added in `production.py`